src/index.html

```diff
@@ -1,67 +1,90 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
this isn't actually in the commit.. I think github is caching the diff in a weird way.

Reply:
fixed